Backend/src/app/controllers/authControllers.ts

```diff
@@ -4,14 +4,30 @@ import { Error_Handler } from "../utils/errorHandling";
 import { User } from "../models/user";
 import jwt from "jsonwebtoken";
 import { sendToken } from "../middlewares/jwtToken";
-import process from "process";
 
 
-export class Auth_Controllers
+enum UserRole
 {
-    public registerUser = asyncError( async (req: Request, res: Response, _next: NextFunction): Promise<void> => {
+    admin = 'admin'
```

Comment:
Missed the default user role?

```suggestion
    Admin = 'admin',
    User = 'user',
```